packages/shared-ux/page/solution_nav/src/solution_nav.test.tsx

```diff
@@ -59,52 +59,43 @@ const items: KibanaPageTemplateSolutionNavProps['items'] = [
   },
 ];
 
-describe('KibanaPageTemplateSolutionNav', () => {
+describe('SolutionNav', () => {
   describe('heading', () => {
     test('accepts more headingProps', () => {
       const component = shallow(
-        <KibanaPageTemplateSolutionNav
-          name="Solution"
-          headingProps={{ id: 'testID', element: 'h3' }}
-        />
+        <SolutionNav name="Solution" headingProps={{ id: 'testID', element: 'h3' }} />
       );
 
       expect(component).toMatchSnapshot();
     });
   });
 
   test('renders', () => {
-    const component = shallow(<KibanaPageTemplateSolutionNav name="Solution" items={items} />);
+    const component = shallow(<SolutionNav name="Solution" items={items} />);
     expect(component).toMatchSnapshot();
   });
 
   test('renders with icon', () => {
-    const component = shallow(
-      <KibanaPageTemplateSolutionNav name="Solution" icon="logoElastic" items={items} />
-    );
+    const component = shallow(<SolutionNav name="Solution" icon="logoElastic" items={items} />);
     expect(component).toMatchSnapshot();
   });
 
   test('renders with children', () => {
     const component = shallow(
-      <KibanaPageTemplateSolutionNav name="Solution" data-test-subj="DTS">
+      <SolutionNav name="Solution" data-test-subj="DTS">
         <span id="dummy_component" />
-      </KibanaPageTemplateSolutionNav>
+      </SolutionNav>
     );
     expect(component.find('#dummy_component').length > 0).toBeTruthy();
   });
 
   test('accepts EuiSideNavProps', () => {
-    const component = shallow(
-      <KibanaPageTemplateSolutionNav name="Solution" data-test-subj="DTS" items={items} />
-    );
+    const component = shallow(<SolutionNav name="Solution" data-test-subj="DTS" items={items} />);
     expect(component).toMatchSnapshot();
   });
 
   test('accepts canBeCollapsed prop', () => {
-    const component = shallow(
-      <KibanaPageTemplateSolutionNav name="Solution" canBeCollapsed={true} items={items} />
-    );
+    const component = shallow(<SolutionNav name="Solution" canBeCollapsed={true} items={items} />);
```

Comment:
Quick nit here but since the default of `canBeCollapsed` is already `true`, I'd guess we'd want to test for the opposite?